README.md

```diff
@@ -1,22 +1,336 @@
-# Diy Json Serializer
+# DIY JSON Serializer
 
 ## Objectives
 
-  1. Understand what JSON is
-  2. Create and Return simple JSON Objects
-  3. Load those objects with AJAX into HTML
-
-## Outline
-
-  * Grab the code from previous readme.
-  * Goal here is to re-do the post show page to be data backed by JS
-  * Previously, we just dumped data out at different routes. This isn't sustainable. We need a way to hit just one route, and get structured data back.
-  * Traditionally this was solved with HTML. HTML has both data and formatting, not good
-  * Just want pure data, used to be given via XML. XML was annoying
-  * Then came JSON
-  * Show them what JSON looks like. Just a dictionary or array of dictionaries!
-  * JSON is a way of "serializing" data. Remember we need to send everything as strings. Serialization means convert from the ruby format of an object, to a string format of an object. 
-  * Let's write our own JSON post serializer.
-  * Make it a separate class with a single class method that takes in the post object and returns a string of JSON
-  * don't use `to_json` or anything like that. Have them by hand build up a JSON dictionary of the important bits of the post
-  * The show page, make it backed by JSON.
+  1. Understand what JSON is.
+  2. Create and Return simple JSON Objects by hand.
+  3. Load those objects with AJAX into HTML.
+
+## Lesson
+
+Continuing with our blog application, we'll be improving our blog post
+API to provide more structure and more efficient data access. Make sure
+to run `rake db:seed` to set up some starting data!
+
+Last time, we created an endpoint for the single piece of data we wanted
+to access. That worked well enough, but doesn't scale. If we want to use
+AJAX to get all the data from a blog post, we'd be creating so many
+endpoints, and making so many GET requests.
+
+What we really need is one endpoint to make one request and get back
+structured data representing our blog post.
```

Comment:
for me, "structured data" is a self-defining term. I'm happy to try to come up with something, but everything I can think of boils down to "data that has a structure".

To that point, I think I need a little clarification on the tone of stuff like this. When I was teaching at Flatiron, the idea was not to spoon-feed every little thing to the students. Rather, we wanted to teach them how to learn, not just facts, which sometimes meant googling something if you didn't know what it was.

I think defining every term that someone may not understand runs the danger of not only bogging down the flow of the lessons, but also training people to start "skimming" things when they think they know it because we keep over-explaining.

so, happy to add some more context to this (and would love any suggestions about defining it) but also think that we might be skewing a little too hand-holdy with something like this (but also, I only know what I knew from before, so feel free to say "hey just do the thing!")